pkg/activator/revision.go

```diff
@@ -153,18 +154,41 @@ func (r *revisionActivator) getRevisionEndpoint(revision *v1alpha1.Revision) (en
 	}, nil
 }
 
-func (r *revisionActivator) ActiveEndpoint(namespace, configuration, name string) (end Endpoint, status Status, err error) {
+func (r *revisionActivator) ActiveEndpoint(namespace, name string) ActivationResult {
 	key := fmt.Sprintf("%s/%s", namespace, name)
 	logger := r.logger.With(zap.String(logkey.Key, key))
-	revision, err := r.activateRevision(namespace, configuration, name)
+	revision, err := r.activateRevision(namespace, name)
 	if err != nil {
 		logger.Error(err)
-		return end, http.StatusInternalServerError, err
+		return ActivationResult{
```

Comment:
We can add configuration and service for error cases?

Reply:
Can't do in the first if block since there is no revision to get that information from, but will do it for the second if block.